RFS/src/main/java/com/rfs/common/RestClient.java

```diff
@@ -34,7 +34,7 @@ public RestClient(ConnectionDetails connectionDetails) {
     }
 
     public Response get(String path, boolean quietLogging) throws Exception {
```

Comment:
Would this be clearer if you passed an Slf4j.Level in here instead.  Why would you want the caller to dictate the log level?
Why is the rest client even logging at all?  I'd recommend that most of the logging in this PR get lifted further up closer to the top of the application rather than in lower-level libraries.  It's fine to have for quick and dirty debugging, but things like errors/warnings will probably be (& should be) big impacts to the overall program, so it would make sense for them to pull all of the context together to show the ramifications.

Reply:
That makes sense; I'll think about that for future iterations.